modules/PROGENY.py

```diff
@@ -0,0 +1,288 @@
+from settings import Config
+from common.HGNCParser import GeneParser
+import sys
+import logging
+import datetime
+import opentargets.model.core as opentargets
+import opentargets.model.bioentity as bioentity
+import opentargets.model.evidence.core as evidence_core
+import opentargets.model.evidence.linkout as evidence_linkout
+import opentargets.model.evidence.association_score as association_score
+
+__copyright__ = "Copyright 2014-2018, Open Targets"
```

Comment:
as above, I'd recommend getting rid of these `__soontobeobsoletedocumentation__` strings.